rekognition-apis/1-object-detection.ipynb

```diff
@@ -85,7 +91,7 @@
     "# We will only use this directory to download images from S3 bucket and draw bounding boxes\n",
     "\n",
     "!mkdir m1tmp\n",
-    "tempFolder = 'm1tmp/'"
+    "tempFolder = 'm1tmp'"
```

Comment:
This variable seems to be unused, perhaps we can just delete this line?

```suggestion
```

Reply:
Yes, we can delete this variable as it is unused